exporters/trace/jaeger/jaeger.go

```diff
@@ -78,15 +80,24 @@ func RegisterAsGlobal() Option {
 	}
 }
 
+func WithDisabled(disabled bool) Option {
+	return func(o *options) {
+		o.Disabled = disabled
+	}
+}
+
 // NewRawExporter returns a trace.Exporter implementation that exports
 // the collected spans to Jaeger.
+//
+// It will IGNORE Disabled option.
 func NewRawExporter(endpointOption EndpointOption, opts ...Option) (*Exporter, error) {
 	uploader, err := endpointOption()
 	if err != nil {
 		return nil, err
 	}
 
 	o := options{}
+	opts = append(opts, WithDisabledFromEnv(), WithProcessFromEnv())
```

Comment:
Should `WithDisabledFromEnv()` be removed if this is supposed to ignore the Disabled option?